test/e2e/federatedresourcequota_test.go

```diff
@@ -125,11 +125,10 @@ var _ = ginkgo.Describe("FederatedResourceQuota auto-provision testing", func()
 				clusterClient, err := util.NewClusterClientSet(clusterName, controlPlaneClient, nil)
 				gomega.Expect(err).ShouldNot(gomega.HaveOccurred())
 
-				gomega.Eventually(func(g gomega.Gomega) (bool, error) {
```

Comment:
?

Reply:
Sorry, this is the code for the test, I will remove it.